src/ISO3166.php

```diff
@@ -15,13 +15,13 @@
 final class ISO3166 implements \Countable, \IteratorAggregate, ISO3166DataProvider
 {
     /** @var string */
-    const KEY_ALPHA2 = 'alpha2';
+    public const KEY_ALPHA2 = 'alpha2';
```

Comment:
```suggestion
    private const KEY_ALPHA2 = 'alpha2';
```

Reply:
I think private makes more sense since these are used internally only?

Reply:
Ah, I suppose you do have a point there. Do you reckon adding the type hints in the previous PR and making these private instead would justify a new major tag? Or would it be quite an inconvenience?

Reply:
hmmm we could add type declarations in https://github.com/thephpleague/iso3166/blob/main/src/ISO3166DataProvider.php that may also justify the major tag, I can probably take a look at that and open a PR later.